<launch>
	<arg name="color_img" value="/color_image" />
	<arg name="mask_img" value="/mask_image" />
	<node name="line_detector" pkg="line_guide" type="detector.py" output="screen" />
	<node name="line_color" pkg="image_view" type="image_view" output="screen">
		<remap from="image" to="$(arg color_img)" />
	</node>
	<node name="line_mask" pkg="image_view" type="image_view" output="screen">
		<remap from="image" to="$(arg mask_img)" />
	</node>
</launch>
